### PR TITLE
Merge 2.x -> 5.1.x

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by Magisus** [comment]
Going to push this directly.

File: project.clj [hunk 1]
```diff
@@ -212,11 +212,12 @@
                                                [puppetlabs/puppetserver ~ps-version :exclusions [puppetlabs/jruby-deps]]
                                                [puppetlabs/trapperkeeper-webserver-jetty9 nil]
                                                [org.clojure/tools.nrepl nil]]
-                      :plugins [[puppetlabs/lein-ezbake "1.8.1"]]
+                      :plugins [[puppetlabs/lein-ezbake "1.1.13"]]
```
Comment (justinstoller) [inline project.clj]: this shouldn't be taken up in 5.x

**Comment by justinstoller** [comment]
why?

**Comment by lindboe** [comment]
I said it was fine with me if she thought it was straightforward, I'm used to not typically reviewing mergeups

**Comment by Magisus** [comment]
Apparently this needed a review, haha. Sorry about that @justinstoller I'll put up something to fix.

**Pull request closed without merging**